doc/tests.rst

```diff
@@ -48,36 +49,38 @@ exec`` to run the test executable (for the sake of this example,
 
 .. code:: bash
 
-   dune exec project/tests/myTest.exe
+dune exec project/tests/myTest.exe
```

Comment:
The indention here is intentional. It's needed for sphinx to display this line as code.

Reply:
Thank you for catching that!

Reply:
Okay! I think I fixed it!